<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderedObject</class>
 <widget class="QWidget" name="RenderedObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>25</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>240</width>
    <height>25</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>240</width>
    <height>25</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="pb_show_hide">
   <property name="geometry">
    <rect>
     <x>175</x>
     <y>3</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: #3D8E4C;
    border: 2px solid #3D8E4C;
	color: rgb(255, 255, 255);
    border-radius: 4px;
}

QPushButton:pressed {
	background: rgb(255, 255, 255);
	border: 2px solid rgb(255, 255, 255);
	color: #3D8E4C;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>h</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_remove">
   <property name="geometry">
    <rect>
     <x>205</x>
     <y>3</y>
     <width>25</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: #3D8E4C;
    border: 2px solid #3D8E4C;
	color: rgb(255, 255, 255);
    border-radius: 4px;
}

QPushButton:pressed {
	background: rgb(255, 255, 255);
	border: 2px solid rgb(255, 255, 255);
	color: #3D8E4C;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_filename">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
